source/code/source/playfab/PlayFabEventBuffer.cpp

```diff
@@ -27,8 +27,8 @@ namespace PlayFab
         bufferArray(std::unique_ptr<uint8_t[]>(new uint8_t[buffMask + 1])),
         buffStart((uint64_t)(bufferArray.get())),
         buffEnd(buffStart + buffMask + 1),
-        eventIndex(std::shared_ptr<std::atomic<uint64_t>>(new std::atomic<uint64_t>(0))),
-        disabled(false)
+        disabled(false),
+        eventIndex(std::shared_ptr<std::atomic<uint64_t>>(new std::atomic<uint64_t>(0)))
```

Comment:
>std::shared_ptr<std::atomic<uint64_t>>(new std::atomic<uint64_t>(0)) [](start = 19, length = 68)

nit: this is unnecessarily verbose.

recommend `std::make_shared<std::atomic_uint64_t>(0)`